jte-models/src/main/java/gg/jte/models/generator/ModelGenerator.java

```diff
@@ -23,17 +23,27 @@ public class ModelGenerator {
     private final String targetClassName;
     private final String interfaceName;
 
+    private final Language language;
+
     public ModelGenerator(TemplateEngine engine, String templateSubDirectory, String targetClassName, String interfaceName) {
```

Comment:
I kept this constructor for compatibility reasons. But it may be unnecessary now since it is not used in the code base.

Reply:
ModelGenerator is internal implementation, so it is not necessary to keep the constructor overload.

Reply:
Removed in aac2a43.